### ISSUE TITLE
Survey

### ISSUE THREAD

**Issue by vhutheesing** [BIDS-collaborative/the-carpool-project#21]
Survey a sample of car owners by asking them a series of questions related to carpooling.  The survey would ask them to select the circumstances under which they would be willing to use a mobile app that enables them to offer rides to others, provided they never have to change their routes or their timings.  These circumstances would include questions on the following considerations and perhaps others:

1) driver safety
2) driver preference
3) driver cancel-ability
4) rider accountability
5) rider payment
6) rider review-ability

Any others?

Joseph, will you lead this exercise with help from me, others in our group, and Dilek?  Dilek from the Air-Quality project has already created and run a survey (see link).

https://berkeley.qualtrics.com/jfe/form/SV_2lgCKBD82nT1Mrj

We will present our survey results and their implications on our model in class on 12/3

Best!

**Comment by reginacpp** [comment]
1. Driver safety: 
   Gender, age, picture, background check, model and year of car, rider's rating, number of rides taken
2. Driver preference: 
   Gender? Age of the car? If rider is willing to be the only rider?
3. Driver cancel-ability:
   Is cancellation free? What are the terms of the penalty?
4. Rider accountability:
   Rider's cancellation rate and acceptance rate?
5. Rider payment:
   Would you have to be paid in advance? Are you willing to be paid on a per-mile basis? Would you be interested in being paid for an entire month?
6. Rider review-ability:
   Rider no-show? Hygiene/odor of rider? Rider too talkative? Unexpected baggage? 
7. Driver review-ability:
   Cleanliness of the car? Functionality of seat belt? Quality of the driving? If driver has dropped the rider off at the requested location?

**Comment by sherlockjjj** [comment]
[commute_survey.docx](https://github.com/BIDS-collaborative/the-carpool-project/files/35226/commute_survey.docx)
First draft.

**Comment by vhutheesing** [comment]
These are good general questions about what people currently do (first section)  The questions also ask what they'd be willing to do (second section), but offer no details about what we are planning to offer.  I think we need a third section that surveys their take on our future service, no?  For example, in your section 2 they may very well say they are not willing to go out of their way to pick up riders, but we still need to ask them if they would be amenable to some aspect of our future envisioned service, right?

Good work.

Vivek

---

rBlock Connects Communities
http://vivekhutheesing.com/
http://rBlock.com

**Comment by sherlockjjj** [comment]
"but we still need to ask them if they would be amenable to some aspect of our future envisioned service", I don't quite understand this sentence.

**Comment by sherlockjjj** [comment]
I think the purpose of this survey is to explore people's willingness to carpool and people's demand for carpool. I think showing them our service would seem hasty.

**Comment by vhutheesing** [comment]
Yes but "carpool" is a word that has lots of connotations in people's minds that may not apply to our service and their assessment of it.  So I suggest we add some what-if questions as well.  For example, "If you answered blank to question blank, what if you never had to go out of your way?  Would this make you amenable to carpooling?  Yes - No - Maybe

**Comment by vhutheesing** [comment]
@sherlockjjj just making some revisions and adding questions.....

**Comment by sherlockjjj** [comment]
Ok, I'll add more questions.

**Comment by vhutheesing** [comment]
@sherlockjjj   I will be at BIDS from 3-5pm today in case you can join me for part of it, but i must leave at 5pm sharp. Great if we can make the survey an online one with Dilek's help. Please read Project Description on this page since I just updated it: http://hackingmeasurement.berkeley.edu/projects/the-carpool-project.html
